src/Applications/LBAF.py

```diff
@@ -379,31 +382,37 @@ def get_output_file_stem(params, n_ranks):
         "initial sent volumes",
         logger=lgr)
 
-    # Prepare input data for rank order enumerator
-    objects = []
-    for rank in phase.get_ranks():
-        for o in rank.get_objects():
-            entry = {
-                "id": o.get_id(),
-                "time": o.get_time(),
-                "to": {},
-                "from": {}}
-            comm = o.get_communicator()
-            if comm:
-                for k, v in comm.get_sent().items():
-                    entry["to"][k.get_id()] = v
-                for k, v in comm.get_received().items():
-                    entry["from"][k.get_id()] = v
-            objects.append(entry)
-    objects.sort(key=lambda x: x.get("id"))
-
-    # Execute rank order enumerator and fetch optimal arrangements
-    n_a, w_min_max, a_min_max = roe.compute_min_max_arrangements_work(
-        objects)
-    if n_a != n_ranks ** len(objects):
-        lgr.error("Incorrect number of possible arrangements with repetition")
-        sys.exit(1)
-    lgr.info(f"Minimax work: {w_min_max:.4g} for {len(a_min_max)} optimal arrangements amongst {n_a}")
+    if params.brute_force_optimization:
+        lgr.info(f"Starting brute force optimization")
```

Comment:
No this is not what I meant: what I meant is that we want to indicate when we do **Not** do it.

When we do it there is no need to indicate it, because we do it from the brute-force results that are shown later in the log :D

Reply:
I added both, should I remove the one when it's on?